core/designsystem/src/main/java/com/iteneum/designsystem/components/Divider.kt

```diff
@@ -0,0 +1,17 @@
+package com.iteneum.designsystem.components
+
+import androidx.compose.material3.Divider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.iteneum.designsystem.theme.LeasePertTheme
+
```

Comment:
Please, add the documentation to this function.

Reply:
documentation added